exercises/03.assertions/01.solution.custom-matchers/README.mdx

```diff
@@ -44,6 +44,16 @@ There are two interfaces from `vitest` that I extend using module augmentgation:
 - `Assertion<T>`, which controls the matchers returned by calling `expect()`;
 - `MatchersDeclaration`, which annotates the matcher declarations passed to `expect.extend()`.
 
+<callout-info>Since Vitest 3.2, you can extend the `Matchers` interface to have type-safe assertions in `expect.extend`, `expect().*` and `expect.*` methods at the same time.</callout-info>
```

Comment:
The exercise text was written before Vitest 3.2 was released. Let's update the text to reflect the proper way of extending the matchers.

Reply:
Makes sense. It would require multiple changes, as other exercises are still referencing the old vitest version.

Reply:
I will go through the remaining mentions of this pattern throughout the workshop and fix them!